.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Add the logs files for NodeJs

Reply:
Juan, la idea inicial de este PR es que monten la primera parte del ambiente, es decir, mocha, esperaba hacer en una sesión la parte de refinar un poco el proyecto y mostrar el gitignore.io, no se que pienses al respecto